man/mkosi.1

```diff
@@ -7,19 +7,19 @@
 mkosi \[em] Build Bespoke OS Images
 .SH SYNOPSIS
 .PP
-\f[C]mkosi [options\&...] build\f[R]
-.PP
-\f[C]mkosi [options\&...] clean\f[R]
-.PP
 \f[C]mkosi [options\&...] summary\f[R]
```

Comment:
Typo in the commit message for the regeneration commit and the middle commit also regenerates this file.

Reply:
Typo is fixed. The regeneration is done twice on purpose so that it's possible to review the actual changes in the second commit.